docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-runtime.adoc

```diff
@@ -2,83 +2,58 @@
 
 Review any changes planned in the next Prisma Cloud release to ensure the security of your runtime.
 
-Read this section to learn about what is planned in the upcoming `32.03` release on the Runtime Security tab of the Prisma Cloud console for WAAS, Host Security, Serverless Security, and Container Security.
+There are no previews or look ahead announcements for the upcoming `32.03` release. Details on the updates included in the `32.03` release will be shared in the release notes that accompany the release.
```

Comment:
@anaghapamidi
I do think that having a link from here to the RNs is a good idea, but it can be error-prone.
That's why we usually commented out lines 5-7 when the RNs were ready and uncommented line 9.
@manukumar6 WDYT?